dashboard/config/initializers/markdown_handler.rb

```diff
@@ -6,8 +7,29 @@ module MarkdownHandler
     space_after_headers: true
   }
 
+  # Rewrite youtube iframe elements to use the fallback-player iframe instead
+  class YoutubeRewriter < Redcarpet::Render::HTML
+    def block_html(html)
+      doc = ::Nokogiri::HTML(html)
+      nodes = doc.xpath(%w(youtube youtubeeducation).map{|x| "//iframe[@src[contains(.,'#{x}.com/embed')]]"}.join(' | '))
+      nodes.each do |node|
+        if node['src']
+          regex = /(http:|https:)?\/\/(www\.)?(youtube|youtubeeducation)\.com\/embed\/([^"&?\/ ]{11})/
+          id = node['src'].match(regex)[4]
+          node['src'] = node['src'].sub(
```

Comment:
I originally used sub! here, but it didn't work so I changed it - it seems that Nokogiri overrides the hash-accessor functions, or something. Here's an `irb` comparison:
```
2.2.1 :021 > Nokogiri::HTML('<iframe src="http://www.youtube.com"/>').tap{|x|x.xpath('//iframe').first['src'].sub!('youtube','X')}.css('body').children.to_html
 => "<iframe src=\"http://www.youtube.com\"></iframe>" 

2.2.1 :022 > Nokogiri::HTML('<iframe src="http://www.youtube.com"/>').tap{|x|x.xpath('//iframe').first.tap{|y|y['src'] = y['src'].sub('youtube','X')}}.css('body').children.to_html
 => "<iframe src=\"http://www.X.com\"></iframe>" 
```